domains/auth/Auth.API/API/Services/CryptographyService.cs

```diff
@@ -10,39 +10,44 @@ public class CryptographyService : ICryptographyService
 
     public string EncryptState(AuthState state)
```

Comment:
Either we should move to a generic Encrypt function that encryptes a string and returns a string, similar to decrypt, or decrypt should specifically return an AuthState.

I would lean towards generic Encrypt(string data)